src/firefly/java/edu/caltech/ipac/firefly/server/persistence/IBEUtils.java

```diff
@@ -129,6 +134,14 @@ public static SortInfo getSortInfo(IbeDataSource source) {
             }
 
 
+        } else if (source instanceof AtlasIbeDataSource) {
+
+                // TODO : explain to me what is used for:
+                sortByCols.put("facility_name", "facility_name, instrument_name, band_name, file_type");
+                //TODO what is 'relatedCols' meant for?
+                // relatedCols = "fname";
```

Comment:
did you get your answers to the `todo` questions here?

Reply:
i remember you told me that the related columns are used for artifacts and such, right? The sort info i guess is to be used in table view if query search table is displayed. Isn't it?
I'll clean up the TODOs with the rest of other cleanups.

Reply:
wow.. this is messy.  this logic should have been kept in IbeDataSource instead of a util function with a case statement.

Reply:
Same mess as before, didn't clean this up and i won't for now, i've used the similar code path and implementation as others ibe datasources on purpose.